style.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: teal;
```

Comment:
I think you should use a hex code or RGB instead of using a color name. Using hex code or RGB will give you unlimited options and you can manipulate digits to obtain any color you desire, and this is something using color name can't do.

Thank you😎